spring-boot-project/spring-boot-docs/src/main/asciidoc/appendix-application-properties.adoc

```diff
@@ -276,6 +276,7 @@ content into your application. Rather, pick only the properties that you need.
 	server.tomcat.max-threads=200 # Maximum amount of worker threads.
 	server.tomcat.min-spare-threads=10 # Minimum amount of worker threads.
 	server.tomcat.port-header=X-Forwarded-Port # Name of the HTTP header used to override the original port value.
+	server.tomcat.processor-cache= # The maximum number of idle processors that will be retained in the cache and re-used with a subsequent request. The default is 200. A value of -1 means unlimited.
```

Comment:
This is still a different description than the field javadoc. As indicated previously, they must be identical.

(When a description is a bit more lengthy than usual, we take the first sentence only but that's not the case here).

Reply:
ok, thx for advice